crates/starknet-server/src/api/http/models.rs

```diff
@@ -63,8 +61,8 @@ pub(crate) struct Time {
 }
 
 #[derive(Serialize)]
-pub(crate) struct PredeployedAccount {
-    pub(crate) initial_balance: u128,
+pub(crate) struct SerializableAccount {
+    pub(crate) initial_balance: String,
```

Comment:
There is a type FeltHex, that was is not used for some reason in this struct. It serialises to 0x.....